src/ensembl/static/img/shared/chevron-left.svg

```diff
@@ -2,8 +2,5 @@
 <!-- Generator: Adobe Illustrator 23.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
 <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
 	 viewBox="0 0 28 28" style="enable-background:new 0 0 28 28;" xml:space="preserve">
-<style type="text/css">
-	.st0{fill:#0099FF;}
-</style>
-<polygon class="st0" points="19,21.3 11.7,14 19,6.7 15.3,6.7 8,14 15.3,21.3 "/>
+<polygon class="chevron" points="19,21.3 11.7,14 19,6.7 15.3,6.7 8,14 15.3,21.3 "/>
```

Comment:
Why do we even need a class in this and the other svg?

Reply:
This must have come from the merge but I wonder why it is appearing as if I added 🤔

Reply:
I was using it before when the panels were there. Don't think it is needed anymore.

Reply:
> as we can target it to change the checkmark colour.
We can do it by targeting the parent, the way it's currently done in the `layout/StandardAppLayout.scss` file above. If we aren't targeting a part of the svg shape (as we did with half-blue half-grey eye), then there's no point in having the class attribute. But I don't have particularly strong feelings about it either way.